src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
+@import "./utils/variables.scss";
+@import "./utils/mixins.scss";
+
+* {
+  margin: unset;
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document